README_MACOS.md

```diff
@@ -72,7 +72,9 @@ Build instructions
     cd build
     cmake -G Xcode -DCMAKE_PREFIX_PATH=`brew --prefix qt5`  ..
     # or, if you want to build with supernova:
+    export FFTW3_DIR=`brew --prefix fftw-sc` # indicate where our custom FFTW is
```

Comment:
alternatively, we could add `/usr/local/opt/fftw-sc` to search paths in `cmake_modules/FindFFTW3f.cmake`